frontend/src/components/Thread.vue

```diff
@@ -1,29 +1,45 @@
 <!-- HTML -->
 <template>
-  <div style="margin-top: 50px">
+  <div style="margin-top: 10px">
     <div v-for="thread in threads">
       <div class="thread text-left">
-        <h4 class="heading"> {{ thread.topic }}
-        <!-- <button class="btn btn-default btn-resolve" v-on:click="resolve(thread)">Resolve</button> -->
+        <h4> {{ thread.topic }}
+          <div style="float: right; margin-top: -5px;">
+            <div class="avatar">
+              <img :src="thread.authorPicture" :alt="thread.authorName" v-if="thread.authorPicture"></img>
+              <div class="glyphicon glyphicon-user" v-else></div>
+            </div>
+          </div>
         </h4>
-        <div class="content">
-          {{ thread.content }}
-          <message :threadID="thread.id"></message>
-          <p class="field">
-            <input type="text" class="textbox" placeholder="Reply..." v-model="newMessages[thread.id]"></input>
-          </p>
-          <button class="btn btn-default btn-reply" v-on:click="reply(thread)">Reply</button>
+        <div v-if="true">
+          <hr/>
+          <div class="content">
+            {{ thread.content }}
+            <message :threadID="thread.id"></message>
+            <input type="text" class="textbox" placeholder="Reply..." style="margin-left: 0px;" v-model="newMessages[thread.id]"></input>
+            <button class="btn btn-default btn-reply" v-on:click="reply(thread)">Reply</button>
+          </div>
         </div>
       </div>
     </div>
-    <div class="thread text-left">
-      <input type="text" class="textbox" placeholder="New thread topic" v-model="newThreadTopic"></input>
-      <input type="text" class="textbox" placeholder="New thread contents" v-model="newThreadContent"></input>
-      <h4 class="heading create-thread hidden-hover" v-on:click="createThread()">
-        <div class="glyphicon glyphicon-plus"></div>
-        Create new thread
-      </h4>
-    </div>
+    <!-- Make this a modal? -->
```

Comment:
Do we still need this comment?

Reply:
Ops, nope not needed anymore. I shall remove.